File: EntityFX.NetBenchamarks.Core/Generic/CallBenchmark.cs
using System.Diagnostics;
using System.Runtime.CompilerServices;

namespace EntityFX.NetBenchmark.Core.Generic
{
    public class CallBenchmark : BenchmarkBase, IBenchamrk
    {
        public CallBenchmark()
        {
            Iterrations = 5000000000;
        }

        [MethodImpl(MethodImplOptions.NoInlining)]
        private static long DoCall(long i)
        {
            return i + 1;
        }

        public override BenchResult Bench()
        {
            var sw = new Stopwatch();
            sw.Start();
            long i = 0;
            long a = 0;
            for (i = 0; i < Iterrations; ++i)
            {
                a = DoCall(i);
            }
            return BuildResult(sw);
        }
    }
}

File: src/php/Generic/CallBenchmark.php
<?php

namespace EntityFX\NetBenchmark\Core\Generic {

    use EntityFX\NetBenchmark\Core\Writer;
    
    class CallBenchmark extends BenchmarkBase {

        public function __construct($writer, $printToConsole)
        {
            parent::__construct($writer, $printToConsole);
            $this->Iterrations = 2000000000;
            $this->Ratio = 0.5;
        }

        private static function DoCall($i)
        {
            return $i + 1;
        }

        protected function BenchImplementation()
        {
            $a = 0;

            for ($i = 0; $i < $this->Iterrations; $i++)
            {
                $a += self::DoCall($i);
            }
            return $a;
        }
    }
}